build/lib/helm_build.sh

```diff
@@ -18,18 +18,37 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-export IMAGE_REGISTRY="${1?First argument is image registry}"
-export IMAGE_REPOSITORY="${2?Second argument is image repository}"
-export IMAGE_TAG="${3?Third argument is image tag}"
-export OUTPUT_DIR="${4?Fourth arguement is output directory}"
-export CHART_NAME=$(basename ${IMAGE_REPOSITORY})
+HELM_REPOSITORY="${1?First argument is helm repoistory}"
+HELM_DIRECTORY="${2?Second argument is helm directory}"
+OUTPUT_DIR="${3?Third arguement is output directory}"
 
-HELM_TEMP_DIR=${OUTPUT_DIR}/helm/${CHART_NAME}
+CHART_NAME=$(basename ${HELM_REPOSITORY})
+DEST_DIR=${OUTPUT_DIR}/helm/${CHART_NAME}
+SOURCE_DIR=$(basename ${HELM_REPOSITORY})/${HELM_DIRECTORY}/.
 
-mkdir -p ${OUTPUT_DIR}/helm/${CHART_NAME}
-cp ${OUTPUT_DIR}/ATTRIBUTION.txt ${HELM_TEMP_DIR}/
-cp -r helm/${CHART_NAME}/. ${HELM_TEMP_DIR}
-envsubst <helm/Chart.yaml.template >${HELM_TEMP_DIR}/Chart.yaml
-envsubst <helm/values.yaml.template >${HELM_TEMP_DIR}/values.yaml
-cd ${HELM_TEMP_DIR}
-helm package .
+#
+# Copy
+#
+mkdir -p ${DEST_DIR}
+cp ${OUTPUT_DIR}/ATTRIBUTION.txt ${DEST_DIR}/
+cp -r ${SOURCE_DIR} ${DEST_DIR}
+
+#
+# Search and replace
+#
+SEDFILE=${OUTPUT_DIR}/helm/sedfile
```

Comment:
Are you breaking this out to that helm_template script below?

Reply:
The other script is gone now